app/lib/benchmarks_fixture.rb

```diff
@@ -0,0 +1,35 @@
+class InvalidParameter < Exception; end
+class OutOfBounds < Exception; end
+
+class BenchmarksFixture
+  def initialize
+    @fixture =
+      JSON.load File.open './app/fixtures/benchmarks_and_activities.json'
+  end
+
+  def activities(benchmark_id, args)
```

Comment:
to destructure, we can do: 
```
def activities(benchmark_id, score:, goal:, level:)
...
end
```

Reply:
Good to know for the future. Turns out to not be needed now that I've split the function into two with all required parameters.